dockerfile.template

```diff
@@ -15,21 +15,23 @@ RUN apt-get -qq install -y --no-install-recommends \
 RUN wget -q https://bootstrap.pypa.io/get-pip.py
 RUN python2.7 get-pip.py
 
-RUN pip2 -q install --upgrade pip
-RUN pip2 -q install setuptools
-RUN pip2 -q install azure azure-cli
+RUN python2.7 -m pip -q install pip==9.0.3
```

Comment:
About the change from pip2/3 install to python2/3 -m pip:
I have been consistently running into issues using pip directly in the command line (both with personal projects and here) so I switched to the python module syntax which consistently works.